src/bindgen/utilities.rs

```diff
@@ -57,6 +57,20 @@ impl SynItemFnHelpers for syn::ItemFn {
     }
 }
 
+impl SynItemFnHelpers for syn::ImplItemMethod {
+    fn exported_name(&self) -> Option<String> {
+        self.attrs
+            .attr_name_value_lookup("export_name")
+            .or_else(|| {
+                if self.is_no_mangle() {
+                    Some(self.sig.ident.to_string())
```

Comment:
Does this really work for impl items? Like, does `impl Foo { extern fn bar(&self) { ... } }` really produce a `bar` symbol?